test/unit/commands/log-specs.js

```diff
@@ -0,0 +1,38 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import _ from 'lodash';
+import sinon from 'sinon';
+import ADB from 'appium-adb';
+import os from 'os';
+import AndroidDriver from '../../..';
+
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('commands - logging', function () {
+  let driver;
+  before(function () {
+    driver = new AndroidDriver();
+    driver.adb = new ADB();
+  });
+  describe('getLogTypes', function () {
+    it('should get log types', async function () {
+      const types = await driver.getLogTypes();
+      // all the types should be returned
+      _.xor(['logcat', 'bugreport', 'server'], types).should.eql([]);
```

Comment:
should we so this verification in the same way it is done for ios driver?

Reply:
If we wanted them to be the same, I would go for this way (since it is order independent).

Reply:
we could also import log type names instead of hardcoding them here

Reply:
I personally would rather they be hardcoded here. Otherwise it doesn't really test anything other than the same data is passed around. And it wouldn't catch an erroneous edit of the type, for instance. When a new type is added it is easy enough to add here.

That all being said, this doesn't need to be an e2e test, and that would shorten the feedback cycle, since the unit tests should be run on commit.